app/src/main/kotlin/com/exallium/todoapp/allnotes/AllNotesAdapter.kt

```diff
@@ -26,6 +51,7 @@ class AllNotesAdapter(val model: AllNotesModel,
     private var dataSubscription: Subscription? = null
     private var clickSubject = PublishSubject.create<Int>()
```

Comment:
Adapter gets view holder reference in onBindViewHolder, that should be enough I think.